src/core/Cline.ts

```diff
@@ -65,6 +65,8 @@ export class Cline {
 	private didEditFile: boolean = false
 	customInstructions?: string
 	alwaysAllowReadOnly: boolean
+	alwaysAllowWrite: boolean
+	alwaysAllowExecute: boolean  // Add this line
```

Comment:
nit: remove extraneous comment:
* ```  // Add this line```
```suggestion
	alwaysAllowExecute: boolean
```